src/main/kotlin/no/nav/medlemskap/regler/common/Regel.kt

```diff
@@ -9,7 +9,8 @@ data class Regel(
         val ytelse: Ytelse,
         val operasjon: () -> Resultat,
         val hvisJa: Regel? = null,
-        val hvisNei: Regel? = null
+        val hvisNei: Regel? = null,
+        val hvisUavklart: Regel? = null
```

Comment:
Jeg er litt usikker på om jeg er helt komfortabel med denne.. Det åpner for en ny utgang på alle regler, og vi har jo ikke definert noen hvisUavklart noe sted i regelflytene. Hvordan kan vi fange opp hvis det returneres noe og vi ikke sjekker for det? (Dette problemet er vel strengt tatt ikke nytt, men denne endringen aktualiserer det..)

Reply:
Regel returnerer jo Svar, og svar kan ha verdiene Ja, Nei eller UVAVKLART. Jeg tror ikke vi har definert flyter for både Ja og Nei for alle regler heller for den sakens skyld.